qmail-local.c

```diff
@@ -388,7 +388,7 @@ int *cutable;
   if (qmeexists(fd,cutable)) {
     if (safeext.len >= 7) {
       i = safeext.len - 7;
-      if (!byte_diff("default",7,safeext.s + i))
+      if (memcmp("default",safeext.s + i,7) == 0)
```

Comment:
Why replacing it inline specifically here? Because it was the only occurrence?

This raises my interest on semantics and guidelines to keep for the project:
This means we aim to replace them inline when possible, unless where too many parts would change (and put more chance to break patches)?

Reply:
Because it should have been ```byte_equal()``` from the start. This is the only ```byte_diff()``` so I thought I could replace it with the final value if I touch it anyway.